zanata/zanata-platform
# feat(ZNTA-2297) Editor Activity Panel Backend

JIRA issue URL: https://zanata.atlassian.net/browse/ZNTA-2297

Implement REST API service to provide all project activity data to the React Editor.
Display this data (Translations, Comments, Revisions) in the frontend activity panel.
Add commenting functionality to the frontend activity panel

## QA
Activity panel should display all Translation History data (Translations, Comments, Revisions) 
Comment box should enforce a max length of 500 (same as the reject translation modals comment textarea) and clear itself after posting. The post comment button should be disabled while the comment textarea is empty.
Filter options should filter the history data by either all items, only comments or only revisions.
Copy buttons to the right of each item should copy the items comment or translation.
![activitypanel](https://user-images.githubusercontent.com/7971187/37321123-a6711708-26c2-11e8-8b83-2d8ed33267b0.png)

## Checklist

- JIRA link
- Check target branch
- Make sure all commit statuses are green or otherwise documented reasons to ignore
- QA needs to evaluate against the JIRA ticket
- Changed files and commits make sense for this PR

See [Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines) more for information.

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/zanata/zanata-platform/729)
